src/app/modules/main/chat_section/controller.nim

```diff
@@ -146,4 +146,34 @@ method clearChatHistory*(self: Controller, chatId: string) =
   self.chatService.clearChatHistory(chatId)
 
 method getCurrentFleet*(self: Controller): string =
-  return self.settingsService.getFleet()
\ No newline at end of file
+  return self.settingsService.getFleet()
+
+method addGroupMembers*(self: Controller, chatId: string, pubKeys: string) =
+  let pubKeysSeq = map(parseJson(pubKeys).getElems(), proc(x:JsonNode):string = x.getStr)
```

Comment:
@borismelnik since you will need this on multiple places, add a private proc which will convert json to the seq of strings (and it should be part of the module.nim, so this conversion should be done there and seq of string just passed to a controller and forwarded from here to a service).

Reply:
Agree with Sale here